test/api-security-documentation.oauth2.test.js

```diff
@@ -179,6 +179,28 @@ describe('<api-security-documentation>', function() {
         });
       });
 
+      describe('raml-example-api', () => {
+        let security;
+        let amf;
+
+        before(async () => {
+          amf = await AmfLoader.load(compact, 'raml-example-api');
+          security = amf[0]['doc:encodes'][0]['apiContract:endpoint'][0]['apiContract:supportedOperation'][0]['security:security'][0];
```

Comment:
This should not work or otherwise the "normal" (not compact) model is not loaded correctly.
Also, the AmfLoader should have a function to lookup a security definition from a method (similar to this: https://github.com/advanced-rest-client/api-method-documentation/blob/stage/test/amf-loader.js#L52). The way you have done this is not scalable and not error proof. I was doing similar thing with 1st generation and components and long term it causes a lot of problems.